css/styles.css

```
@@ -0,0 +1,93 @@
+/*
+- Font sizes (px):
+10 / 12 / 14 / 16 / 18 / 20 / 24 / 30 / 36 / 44 / 52 / 62 / 74 / 86 / 98
+
+- Spacing system (px)
+2 / 4 / 8 / 12 / 16 / 24 / 32 / 48 / 64 / 80 / 96 / 128
+
+*/
+
+:root {
+  font: 400 10px / 1.2 "Yeseva One";
+}
+
+.container {
+  width: 160rem;
+  margin: 0 auto;
+}
+
+.header {
+  display: flex;
+  height: 12rem;
+  gap: 0.4rem;
+  padding-left: 18rem;
+  padding-right: 20.4rem;
+  align-items: center;
+}
+
+.logo {
+  font-size: 3.6rem;
+  text-transform: uppercase;
+  margin-right: auto;
+}
+
+.nav {
+  font-family: "Gentium Book Plus";
+  font-weight: bold;
```

Let's use numeric variant for all the weights
```suggestion
  font-weight: 700;
```